### PR TITLE
Wireshark CVEs, and a CVE template, and some CSS fixes

### PR DESCRIPTION
**Pull request by todb**
Why yes, it's a PR with lots of different changes! I'm a naughty, naughty boy!

Here's what this PR does:

* Creates a CVE writing template, useful for future vulnerability disclosures. (#152 takes care of this separately).
* Fixes the CSS so `<h1>` through `<h6>` isn't justified-aligned, because it tends to look real bad.
* Discloses CVE-2023-0666, 0667, and 0668.

On that last thing -- @zenofex and @WanderingGlitch should eyeball these, and ideally so would @geraldcombs. We got caught by surprise a little by the bugs all getting switched to public, but that's not a big deal at all. We can release early and write up the CVEs once these advisories publish.

Also: Not that this is much of a risk, but these shouldn't publish before Monday, June 5. So, no sooner than six hours from now, but before 30 hours from this moment.

### CODE REVIEW

File: cves/CVE-2023-0667.md
Comment (Zenofex) [inline cves/CVE-2023-0667.md:1]: The attacker value header needs a newline before it to display the proper markdown.

File: cves/CVE-2023-0668.md
Comment (Zenofex) [inline cves/CVE-2023-0668.md:1]: The attacker value header needs a newline before it to display the proper markdown.

File: cves/CVE-2023-0667.md
Comment (todb) [reply cves/CVE-2023-0667.md:1]: Fixed!

**Comment by todb** [comment]
Note, there's some time-sensitive data in here. When this lands:

* Make sure the Timeline elements are accurate. If it's Monday, Jun 5, 2023, then no changes are needed. If after, then, fix that before landing.
* Once published, we can file the real CVEs (via https://vulnogram.github.io/#editor). @mauvehed says he wants to do that for practice.

**Review by mauvehed — APPROVED** [review]

**Review by mauvehed — DISMISSED** [review]

**Review by mauvehed — DISMISSED** [review]

**Review by mauvehed — APPROVED** [review]

**Review by mauvehed — APPROVED** [review]
STOP CHANGING SHIT, TOD!

**Comment by todb** [comment]
okay, running into the deploy of this:

https://github.blog/changelog/2023-06-06-security-enhancements-to-required-approvals-on-pull-requests/

I say force it

**Comment by todb** [comment]
I promise, I have not "dismissed" anything even though GitHub is blaming me.

**Pull request merged**